services/keycloak/start.sh

```diff
@@ -1489,10 +1489,21 @@ function configure_problems_system {
 
   echo Creating resource problem
 
-  echo '{"name":"problem","displayName":"problem","scopes":[{"name":"view"},{"name":"add"},{"name":"delete"}],"attributes":{},"uris":[],"ownerManagedAccess":""}' | /opt/jboss/keycloak/bin/kcadm.sh create clients/$CLIENT_ID/authz/resource-server/resource --config $CONFIG_PATH -r ${KEYCLOAK_REALM:-master} -f -
+  echo '{"name":"problem","displayName":"problem","scopes":[{"name":"viewAll"},{"name":"view"},{"name":"add"},{"name":"delete"}],"attributes":{},"uris":[],"ownerManagedAccess":""}' | /opt/jboss/keycloak/bin/kcadm.sh create clients/$CLIENT_ID/authz/resource-server/resource --config $CONFIG_PATH -r ${KEYCLOAK_REALM:-master} -f -
```

Comment:
This is going to need to move into its own function - by the time your code runs, the problems system will have already have run, and therefore this probably won't run again

Reply:
Ah yes - although this is something I wanted to chat about. Is this global check even needed? There are checks on to view each problem and project/environment already.